jerry-core/parser/regexp/re-parser.h

```diff
@@ -35,27 +35,27 @@
 typedef enum
 {
   RE_TOK_EOF,                        /**< EOF */
-  RE_TOK_BACKREFERENCE,              /**< \[0..9] */
+  RE_TOK_BACKREFERENCE,              /**< "\[0..9]" */
   RE_TOK_CHAR,                       /**< any character */
-  RE_TOK_ALTERNATIVE,                /**< | */
-  RE_TOK_ASSERT_START,               /**< ^ */
-  RE_TOK_ASSERT_END,                 /**< $ */
-  RE_TOK_PERIOD,                     /**< . */
-  RE_TOK_START_CAPTURE_GROUP,        /**< ( */
-  RE_TOK_START_NON_CAPTURE_GROUP,    /**< (?: */
-  RE_TOK_END_GROUP,                  /**< ')' */
-  RE_TOK_ASSERT_START_POS_LOOKAHEAD, /**< (?= */
-  RE_TOK_ASSERT_START_NEG_LOOKAHEAD, /**< (?! */
-  RE_TOK_ASSERT_WORD_BOUNDARY,       /**< \b */
-  RE_TOK_ASSERT_NOT_WORD_BOUNDARY,   /**< \B */
-  RE_TOK_DIGIT,                      /**< \d */
-  RE_TOK_NOT_DIGIT,                  /**< \D */
-  RE_TOK_WHITE,                      /**< \s */
-  RE_TOK_NOT_WHITE,                  /**< \S */
-  RE_TOK_WORD_CHAR,                  /**< \w */
-  RE_TOK_NOT_WORD_CHAR,              /**< \W */
-  RE_TOK_START_CHAR_CLASS,           /**< [ ] */
-  RE_TOK_START_INV_CHAR_CLASS,       /**< [^ ] */
+  RE_TOK_ALTERNATIVE,                /**< "|" */
+  RE_TOK_ASSERT_START,               /**< "^" */
+  RE_TOK_ASSERT_END,                 /**< "$" */
+  RE_TOK_PERIOD,                     /**< "." */
+  RE_TOK_START_CAPTURE_GROUP,        /**< "(" */
+  RE_TOK_START_NON_CAPTURE_GROUP,    /**< "(?:" */
+  RE_TOK_END_GROUP,                  /**< ")" */
+  RE_TOK_ASSERT_START_POS_LOOKAHEAD, /**< "(?=" */
+  RE_TOK_ASSERT_START_NEG_LOOKAHEAD, /**< "(?!" */
+  RE_TOK_ASSERT_WORD_BOUNDARY,       /**< "\b" */
+  RE_TOK_ASSERT_NOT_WORD_BOUNDARY,   /**< "\B" */
+  RE_TOK_DIGIT,                      /**< "\d" */
+  RE_TOK_NOT_DIGIT,                  /**< "\D" */
+  RE_TOK_WHITE,                      /**< "\s" */
+  RE_TOK_NOT_WHITE,                  /**< "\S" */
+  RE_TOK_WORD_CHAR,                  /**< "\w" */
+  RE_TOK_NOT_WORD_CHAR,              /**< "\W" */
+  RE_TOK_START_CHAR_CLASS,           /**< "[ ]" */
+  RE_TOK_START_INV_CHAR_CLASS,       /**< "[^ ]" */
```

Comment:
Otherwise doxygen prints warnings like these:
```
/home/akiss/devel/jerry/jerryscript/jerry-core/parser/regexp/re-parser.h:53: warning: Found unknown command `\s'
/home/akiss/devel/jerry/jerryscript/jerry-core/parser/regexp/re-parser.h:56: warning: Found unknown command `\W'
/home/akiss/devel/jerry/jerryscript/jerry-core/parser/regexp/re-parser.h:50: warning: Found unknown command `\B'
/home/akiss/devel/jerry/jerryscript/jerry-core/parser/regexp/re-parser.h:52: warning: Found unknown command `\D'
/home/akiss/devel/jerry/jerryscript/jerry-core/parser/regexp/re-parser.h:55: warning: Found unknown command `\w'
/home/akiss/devel/jerry/jerryscript/jerry-core/parser/regexp/re-parser.h:54: warning: Found unknown command `\S'
/home/akiss/devel/jerry/jerryscript/jerry-core/parser/regexp/re-parser.h:51: warning: Found unknown command `\d'
/home/akiss/devel/jerry/jerryscript/jerry-core/parser/regexp/re-parser.h:44: warning: End of list marker found without any preceding list items
```